app/controllers/smart_answers_controller.rb

```diff
@@ -90,10 +91,6 @@ def redirect_response_to_canonical_url
     end
   end
 
-  def set_header_footer_only
```

Comment:
It looks like this determines the layout of the pages, so I think this is not supposed to be deleted.